static/news.ru.html

```diff
@@ -1,12 +1,29 @@
 <h1 id="helpPage_newsTitle">Новости</h1>
 
+<h2 class="title">Новый режим, регулировка сложности и не только</h2>
+<span class="date">
+    __.08.2020
+</span><span class="version">
+    v1.2
+</span><span class="new">
+    * Новое
+</span>
+<p>
+Играть стало ещё удобнее и интереснее!
+
+<li>Добавили режим игры со словами игроков: каждый участник пишет слова, из которых формируется набор слов в шляпе</li>
+<li>Появились словари разного уровня сложности на русском языке</li>
+<li>Если у вас есть собственный словарь, теперь вы сможете его загрузить и играть со словами из него</li>
+<li>Появилась возможность ограничить не количество слов, а количество кругов</li>
+<li> Отображение количества ходов до вашего следующего участия. Захотели налить чайку? Теперь всё можно спланировать!</li>
```

Comment:
```suggestion
<li>Отображение количества ходов до вашего следующего участия. Захотели налить чайку́? Теперь всё можно спланировать!</li>
```
P.S. Я, честно говоря, сначала прочитал "налить ча́йку".

Reply:
лол

Reply:
Не надо наливать ча́йку, а ударение правда лучше поставить